Makefile

```diff
@@ -17,7 +17,7 @@ generate:
 check: generate
 	cargo check --all --all-targets --features "${ALL_FEATURES}"
 	cargo fmt -- --check
-	cargo clippy --all-targets --features "${ALL_FEATURES}" -- -D clippy::all
+	cargo clippy --fix --allow-dirty --all-targets --features "${ALL_FEATURES}" -- -D clippy::all
```

Comment:
It's not a good idea to add `--fix --allow-dirty`, as the fix will be done in CI only and not applied to code base. Besides, some automatic change may not be correct, we need to have a check.

I will open a PR in these few days to fix the error introduced by the latest toolchain.

Reply:
Thanks @pingyu . It seems like the toolchain upgrade started flagging the autogenerated docs. `--fix --allow-dirty` is a workaround to make the build go through. I did't want to pin the toolchain to a certain version, so this was a quick way to bypass the clippy errors. I agree with you on not using `--fix --allow-dirty`. Do you have any suggestion on how I can get this PR merged? Looks like all the tests are passing.

Reply:
> It's not a good idea to add `--fix --allow-dirty`, as the fix will be done in CI only and not applied to code base. Besides, some automatic change may not be correct, we need to have a check.
> 
> I will open another PR in these few days to fix the errors introduced by the latest toolchain.

The check errors for latest toolchain will be fixed by #460.